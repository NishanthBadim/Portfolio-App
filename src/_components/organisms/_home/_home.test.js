import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Home from "./_home";

jest.mock("../../atoms/_avatar/_avatar", () => ({
    __esModule: true,
    default: () => <div>Avatar mocked</div>
}));
jest.mock("../../molecules/_banner/_banner", () => ({
    __esModule: true,
    default: () => <div>Banner mocked</div>
}));
describe("Home component testing", () => {
    test("Matches the snapshot", () => {
        const input = create(<Home />);
        expect(input.toJSON()).toMatchSnapshot();
    });

});