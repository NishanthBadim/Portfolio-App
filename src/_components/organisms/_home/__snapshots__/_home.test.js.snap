// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing Matches the snapshot 1`] = `
<div
  className="mdl-grid makeStyles-root-1"
>
  <div
    className="mdl-cell mdl-cell--12-col"
  >
    <div>
      Avatar mocked
    </div>
    <div>
      Banner mocked
    </div>
  </div>
</div>
`;
